portfolio/src/main/webapp/script.js

```diff
@@ -12,55 +12,126 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const MAX_COMMENTS = 100;
+
+/*************************
+ *     RANDOM FACT 
+ ************************/
+
 /**
  * Adds a random fact to the page.
  */
-function addRandomFact() {
+function randomFact() {
   const facts = ["My Google Drive contains over 22TB of files",
-   "I'm currently watching The Office for the third time",
-    "I have a turtle as a pet but I haven't seen it in a week"];
+  "I'm currently watching The Office for the third time",
+  "I have a turtle as a pet but I haven't seen it in a week",
+  "Sometimes I worry about the amount of computers that I have.",
+  "I low key hoard electronics and then regret not selling them when they were worth something",
+  "My mom thinks my CS Degree (in progress) gives me magical Facebook powers.",
+  "As a CS student I've been asked if I can hack a Facebook account more than about algorithms."];
 
-  // Pick a random greeting.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
+  const fact = facts[Math.floor(Math.random() * facts.length)]; // Pick a random fact.
 
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+  const factContainer = document.getElementById('fact-container');  
+  factContainer.innerText = fact; // Add it to the page.
 }
 
-function getName(){
-  console.log('Fetching name!');
+/*************************
+ *     Greeting
+ ************************/
+function greetingMaker() {
+  const greetings = ["Bonjour!","Ahoj!","Hola!","Hello!","Guten Tag!", "Yasou!"];
 
-  // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
+  const greeting = greetings[Math.floor(Math.random() * greetings.length)]; // Pick a random greeting.
 
-  // When the request is complete, pass the response into handleResponse().
-  responsePromise.then(handleResponse);
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;  // Add it to the page.
 }
 
+/*************************
+ *     Comments
+ ************************/
+async function queryComments(){
+  const query = await fetch("/comment");
+  const jsonResponse = await Response.json();
 
-/**
- * Handles response by converting it to text and passing the result to
- * addQuoteToDom().
- */
-function handleResponse(response) {
-  console.log('Handling the response.');
+  // Insert comments to HTML
+}
```

Comment:
Is this function used? It looks like maybe not.

Reply:
you're right, I replaced that one with the addComments function...deleting now